.github/workflows/verify-dependabot.yml

```diff
@@ -0,0 +1,50 @@
+name: Verify (dependabot)
+
+on:
+  pull_request_target:
+    branches:
+      - main
+
+env:
+  NODE_VERSION: 14.x
+
+jobs:
+  build:
+    name: Build and Test
+    runs-on: ubuntu-latest
+    if: github.actor == 'dependabot[bot]'
+    steps:
+      - name: Cancel Previous Runs
+        uses: styfle/cancel-workflow-action@0.9.1
+
+      - name: Checkout Code
+        uses: actions/checkout@v2.3.4
+        with:
+          ref: ${{ github.event.pull_request.head.sha }}
+
+      - name: Use Node.js ${{ env.NODE_VERSION }}
+        uses: actions/setup-node@v2.4.0
+        with:
+          node-version: ${{ env.NODE_VERSION }}
+          cache: yarn
+
+      - name: Configure Package Manager
+        run: |
+          echo Configuring NPM_READ_TOKEN globally for .npmrc
+          npm config set '//registry.npmjs.org/:_authToken' ${{ secrets.NPM_READ_TOKEN }}
+          npm whoami
+
+
+      - name: Install Dependencies
+        run: yarn install --frozen-lockfile
+
+      - name: Lint
+        run: yarn lint
+
+      - name: Test
+        run: yarn test --coverage --silent
+        env:
+          GITHUB_TOKEN: ${{ secrets.GITHUB_TOKEN }}
```

Comment:
```suggestion
```

Not needed

Reply:
It's needed to test workflow status retrieval using gh CLI
We discussed about this a few months ago and you said that it's a good idea to not mock this call and instead use the GITHUB_TOKEN env var to be able to test real gh calls without having to specify credentials inside the test code.
If you think it's not good to do so, I can remove the tests related to the gh CLI status retrieval and instead mock this call by returning some values

Reply:
Can you add the same coverage item here like you did for verify?